versionInfo.gradle

```diff
@@ -1,5 +1,20 @@
 group = 'com.workday.warp'
 
+
+println project.findProperty("scalaVersion")
```

Comment:
Is this just print temporary, or did you want to leave this in?

Reply:
ill remove it. just note that the property `scalaVersion` is added by the multiversion plugin i added, and won't be available during buildscript time